theano/gof/params_type.py

```diff
@@ -519,18 +524,14 @@ def c_support_code(self):
         struct_name = self.name
         struct_name_defined = struct_name.upper()
         c_support_code_set = set()
```

Comment:
Make this an orderedset. Otherwise, the code generated could change due to non related causes.